src/gui_common/menus/PlanetCustomizerTool.cs

```diff
@@ -0,0 +1,117 @@
+#pragma warning restore CA2213
+
+    private WorldGenerationSettings worldSettings = null!;
+
+    private GameProperties gameProperties = null!;
+
+    public override void _Ready()
+    {
+        base._Ready();
+        InitNewWorld(SimulationParameters.Instance.AutoEvoConfiguration);
+        patchMapDrawer.OnSelectedPatchChanged += UpdatePatchDetailPanel;
+    }
+
+    private void InitNewWorld(IAutoEvoConfiguration configuration)
+    {
+        var planetGenerationSettings = planetSettings.GetPlanetSettings();
+        worldSettings = new WorldGenerationSettings
+        {
+            AutoEvoConfiguration = configuration,
+            WorldSize = planetGenerationSettings.WorldSize,
+            WorldTemperature = planetGenerationSettings.WorldTemperature,
+            WorldSeaLevel = planetGenerationSettings.WorldSeaLevel,
+            GeologicalActivity = planetGenerationSettings.GeologicalActivity,
+            ClimateInstability = planetGenerationSettings.ClimateInstability,
+            Origin = planetGenerationSettings.Origin,
+            DayNightCycleEnabled = planetGenerationSettings.DayNightCycleEnabled,
+            DayLength = planetGenerationSettings.DayLength,
+            LAWK = planetGenerationSettings.LAWK,
+        };
+
+        gameProperties = GameProperties.StartNewMicrobeGame(worldSettings);
+        gameProperties.GameWorld.Map.RevealAllPatches();
+        patchMapDrawer.PlayerPatch = null;
+        patchMapDrawer.Map = gameProperties.GameWorld.Map;
+        patchMapDrawer.SelectedPatch = patchMapDrawer.PlayerPatch;
+        patchDetailsPanel.SelectedPatch = patchMapDrawer.PlayerPatch;
+    }
+
+    private void UpdatePatchDetailPanel(PatchMapDrawer drawer)
+    {
+        var selectedPatch = drawer.SelectedPatch;
+
+        if (selectedPatch == null)
+            return;
+
+        patchDetailsPanel.SelectedPatch = selectedPatch;
+    }
+
+    private void OnBackButtonPressed()
+    {
+        TransitionManager.Instance.AddSequence(ScreenFade.FadeType.FadeOut, 0.1f,
+            SceneManager.Instance.ReturnToMenu, false);
+    }
+
+    private void OnGeneratePressed()
+    {
+        if (settingsPanel.Visible)
+        {
+            InitNewWorld(SimulationParameters.Instance.AutoEvoConfiguration);
+            generateButton.Text = Localization.Translate("BACK_TO_SETTINGS");
+        }
+        else
+        {
+            generateButton.Text = Localization.Translate("GENERATE_BUTTON");
+        }
+
+        settingsPanel.Visible = !settingsPanel.Visible;
+        patchMapPanel.Visible = !patchMapPanel.Visible;
+        patchMapButtons.Visible = !patchMapButtons.Visible;
+    }
+
+    private void OnRegeneratePressed()
+    {
+        InitNewWorld(SimulationParameters.Instance.AutoEvoConfiguration);
+    }
+
+    private void OnAutoEvoToolPressed()
+    {
+        var scene = SceneManager.Instance.LoadScene("res://src/auto-evo/AutoEvoExploringTool.tscn").Instantiate();
```

Comment:
This doesn't match our usual pattern of `GD.Load<PackedScene>("stuff")` so I'd change this for code consistency purposes.